executors/cpp/coll.cpp

```diff
@@ -217,9 +222,13 @@ auto TestCollator(json_object *json_in) -> string {
   if (compare_type_string == "" || compare_type_string.substr(0, 1) == "<") {
     // Default checking for <= 0.
     coll_result = (uni_result != UCOL_GREATER);
-  } else {
-    coll_result = (uni_result == UCOL_EQUAL);
-  }
+  } else
+    if (compare_type_string == "=") {
+      coll_result = (uni_result == UCOL_EQUAL);
+    } else {
+      //
```

Comment:
Did you forget to fill in a comment here? I assume that we test against UCOL_EQUAL as a default because we don't have any other compare types except `=` and the implicit `<=`, and `=` is a fine enough default. Is that correct?